### ISSUE TITLE
Asiakkaalle mennyt aiheeton s-posti "Kirjastokorttisi PIN-koodi vaihdettiin"

### ISSUE THREAD

**Issue by pirliis** [KohaSuomi/Koha#629]
### Mikä vikana?

OUTIn asiakas (borrowernumber=37472) on ilmoittanut, että hän oli saanut gmail-sähköpostiinsa ilmoituksen, että kirjastokorttisi PIN-koodi vaihdettiin. Asiakkaan kertoman mukaan hän ei ole sitä tehnyt, eikä asiakkaan muutoslokillakaan näy tietoa, että  PIN-koodia olisi vaihdettu asiakkaan tai virkailijan toimesta. Asiakkaan Ilmoitukset-sivulla ei ollut myöskään ko. ilmoitusta. Asiakkaan huollettavalle, jolla sama s-postiosoite kuin huoltajalla, ei ole PIN-koodia vaihdettu. Tietokannasta ei löydy muita asiakkaita, joilla olisi sama s-postiosoite kuin tällä henkilöllä.

Kuvakaappaus asiakkaalle menneestä viestistä:
![image](https://github.com/KohaSuomi/Koha/assets/98814860/bbb03ff5-f8bf-4b11-871a-e7e2e82ef973)

Vastaanottaja on ollut asiakkaan tiedoissa näkyvä sähköpostiosoite. 
Pystyykö selvittämään, miksi asiakkaalle on ko. viesti mennyt, vaikka PIN-koodia ei ole vaihdettu ainakaan omien selvitystemme perusteella?

**Comment by lmstrand** [comment]
Asiakkaan tietoja on tietokannan mukaan muutettu viimeksi:

 updated_on: 2023-06-02 13:57:38

**Comment by pirliis** [comment]
Kyllä. Kun asiakas on ottanut yhteyttä kirjastoon 2.6.2023 on varotoimenpiteenä laitettu asiakkaan kortti kadonneeksi ja laitettu huomautusviesti. Asiakkaalla on kortti tallessa, mutta olivat virkailijan kanssa sopineet, että hän käy kirjastossa vaihtamassa sen uuteen kaiken varalta.

**Comment by AnneliO** [comment]
Tuo viesti on lähtenyt tälle asiakkaalle: https://outi.koha-suomi.fi/cgi-bin/koha/members/moremember.pl?borrowernumber=144521

Hänellä on aiemmin ollut käytössä tuon asiakkaan spostiosoite ja se on vaihdettu asiakkaan omaksi samana päivänä kuin salasanakin on vaihdettu.

Olettaisin kyseessä olevan asiakkaan aikuistuneen lapsen, koska osoite on sama ja hän on täyttänyt 18 keväällä. :)
